src/appleseed/renderer/modeling/environmentedf/oslenvironmentedf.cpp

```diff
@@ -99,12 +204,23 @@ namespace
             OnFrameBeginRecorder&   recorder,
             IAbortSwitch*           abort_switch) override
         {
+
             if (!EnvironmentEDF::on_frame_begin(project, parent, recorder, abort_switch))
                 return false;
-
+ 
             m_shader_group =
                 static_cast<ShaderGroup*>(m_inputs.get_entity("osl_background"));
 
+            if(m_importance_sampler.get() == nullptr || m_shader_group->get_version_id() != m_shader_group_version_id){
```

Comment:
Replace `m_importance_sampler.get() == nullptr` by `!m_importance_sampler`.

Insert space after `if`, move opening brace to its own line.

You can find our coding guidelines on the wiki:
https://github.com/appleseedhq/appleseed/wiki/Coding-Philosophy-and-Guidelines